app/domain/authentication/authn_k8s/k8s_host.rb

```diff
@@ -38,9 +45,9 @@ def initialize(account:, service_name:, common_name:)
       end
 
       def conjur_host_id
-        host_id = "#{@account}:" + @common_name.k8s_host_name.sub('host/', 'host:')
-        Rails.logger.debug(Log::HostIdFromCommonName.new(host_id))
-        host_id
+        "#{@account}:#{@common_name.k8s_host_name.sub('host/', 'host:')}".tap do |host_id|
+          Rails.logger.debug(Log::HostIdFromCommonName.new(host_id))
+        end
```

Comment:
I prefer the original in this case: it's more straightforward.  But I don't mind this.

Reply:
i added a question about this in slack: https://conjurhq.slack.com/archives/C607Z50D6/p1589191597004900

This is not the only occurrence of this so we need to decide on a guideline

Reply:
we'll get this aligned in a separate PR.

Reply:
Agreed on preferring the original. `tap` is complex enough to rarely be worth including.